Library/src/main/java/info/metadude/android/typedpreferences/BooleanPreference.java

```diff
@@ -1,28 +1,52 @@
+/*
+ * Copyright (C) 2020-21 Application Library Engineering Group
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
 package info.metadude.android.typedpreferences;
 
-import android.content.SharedPreferences;
+import ohos.data.preferences.Preferences;
 
 /**
  * A wrapper class for a boolean preference.
  */
 public class BooleanPreference extends BasePreference {
 
     protected final boolean mDefaultValue;
+
     public static final boolean DEFAULT_VALUE_VALUE = false;
 
     /**
-     * Constructs a {@code boolean} preference for the given key
-     * having the default value set to {@code false} available.
+     * BooleanPreference.
```

Comment:
Retain the base library's java doc comments for all the methods as its more meaningful. Applies to all the methods.